api/src/test/java/org/pmiops/workbench/db/dao/UserServiceTest.java

```diff
@@ -82,21 +86,21 @@
   private static DbAccessTier registeredTier;
   private static List<DbAccessModule> accessModules;
 
-  @MockBean private FireCloudService mockFireCloudService;
   @MockBean private ComplianceService mockComplianceService;
   @MockBean private DirectoryService mockDirectoryService;
-  @MockBean private UserServiceAuditor mockUserServiceAuditAdapter;
-  @MockBean private UserTermsOfServiceDao mockUserTermsOfServiceDao;
```

Comment:
changed this one to autowired because I need its functionality.

alphabetized each section.